Doppler.HtmlEditorApi/Storage.DapperProvider/Queries/DeleteRemovedCampaignLinks.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+
+namespace Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
+
+/// <summary>
+/// It deletes DB entries that are not present in the payload.
+/// It does not insert or update nothing else.
+/// </summary>
+/// <remarks>
+/// Dapper deals with the SQL `IN` operator and the IEnumerable.
+/// (See https://github.com/DapperLib/Dapper#list-support)
+/// Example with 4 links:
+///     exec sp_executesql N'
+///       DELETE FROM [Link]
+///       WHERE [Link].IdCampaign = @IdContent
+///       AND [Link].UrlLink NOT IN (@Links1,@Links2,@Links3,@Links4)',
+///     N'@IdContent int,@Links1 nvarchar(4000),@Links2 nvarchar(4000),@Links3 nvarchar(4000),@Links4 nvarchar(4000)',
+///     @IdContent=36357513,@Links1=N'https://www.google.com?q=|*|320*|*',@Links2=N'a',@Links3=N'b',@Links4=N'c'
+/// Example without links:
+///     exec sp_executesql N'
+///       DELETE FROM [Link]
+///       WHERE [Link].IdCampaign = @IdContent
+///       AND [Link].UrlLink NOT IN (SELECT @Links WHERE 1 = 0)',
+///     N'@IdContent int,@Links nvarchar(4000)',
+///     @IdContent=36357513,@Links=NULL
```

Comment:
@MarianoHerrera maybe you are interested on review these queries.

Reply:
The example isn't acurate you have a list of links and not a "select" with "1=0" (never will return anithing)

Reply:
Right, so, in this case, it should delete all the links with `IdCampaign = @IdContent`, so, it is fine.